gatherpress.php

```diff
@@ -52,3 +52,38 @@ function() {
 
 require_once GATHERPRESS_CORE_PATH . '/core/loader.php';
 require_once GATHERPRESS_CORE_PATH . '/buddypress/loader.php';
+
+register_activation_hook( __FILE__, 'activate_gatherpress_plugin' );
+/**
+ * Activate GatherPress plugin.
+ *
+ * @return void
+ */
+function activate_gatherpress_plugin() {
+	GatherPress\Core\Setup::register_post_types();
+	if ( ! get_option( 'gatherpress_flush_rewrite_rules_flag' ) ) {
+		add_option( 'gatherpress_flush_rewrite_rules_flag', true );
```

Comment:
No, option is fine, but thinking we store the plugin version (eg 0.5 right now) from the constant rather than true/false, and when it changes because of an update to the plugin we flush rewrite rules and store the the version again in options. just call the option `gatherpress_version` or something like that.